app/views/families/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <body>
```

Comment:
You don't need to set all those tags, because they are defined in `app/views/layouts/application.html.erb`
If you look at the families index view in the browser, I believe you will see those tags defined twice.

Reply:
👍